### PR TITLE
Updates City Guide BMW Logo

### PR DESCRIPTION
**Pull request by ashleyjelks**
https://artsyproduct.atlassian.net/browse/FX-1876

Simulator Screen Shot - iPhone 11 - 2020-04-17 at 15 36 43
Simulator Screen Shot - iPhone 11 - 2020-04-17 at 15 37 01
Simulator Screen Shot - iPhone 11 - 2020-04-17 at 16 14 47
Simulator Screen Shot - iPhone 11 Pro Max - 2020-04-17 at 15 35 13
Simulator Screen Shot - iPhone 11 Pro Max - 2020-04-17 at 15 36 23

#skip_new_tests

### CODE REVIEW

**Comment by ashfurrow** [comment]
@ashleyjelks since you're in here anyway, I was wondering if I could ask a favour? This text is not vertically aligned with the icon, and it's bothered me forever but I've never gotten around to fixing it.

Screen Shot 2020-04-17 at 17 01 45

Would you mind taking a look? No worries if it's out of scope or anything!

**Comment by ashleyjelks** [comment]
> @ashleyjelks since you're in here anyway, I was wondering if I could ask a favour? This text is not vertically aligned with the icon, and it's bothered me forever but I've never gotten around to fixing it.
> 
> Screen Shot 2020-04-17 at 17 01 45
> 
> Would you mind taking a look? No worries if it's out of scope or anything!

No problem, can you screenshot the entire screen so I know where to make that change? This view appears in multiple places?

**Comment by ashleyjelks** [comment]
> > @ashleyjelks since you're in here anyway, I was wondering if I could ask a favour? This text is not vertically aligned with the icon, and it's bothered me forever but I've never gotten around to fixing it.
> > Screen Shot 2020-04-17 at 17 01 45
> > Would you mind taking a look? No worries if it's out of scope or anything!
> 
> No problem, can you screenshot the entire screen so I know where to make that change? This view appears in multiple places?

Actually, nvm, I know where that is!

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Missing Test Files:

- `src/lib/Scenes/City/__tests__/CityBMWSponsorship-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "#skip_new_tests".</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: src/lib/Scenes/City/CityBMWSponsorship.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import React, { useRef } from "react"
+import { TouchableOpacity } from "react-native"
+import { useTracking } from "react-tracking"
+import styled from "styled-components/native"
+
+interface BMWSponsorshipProps {
+  url?: string
+  logoText: string
+  mt: number
+  ml: number
```
Comment (xtina-starr) [inline src/lib/Scenes/City/CityBMWSponsorship.tsx:13]: `string` is also a valid type/value for `ml` and `mt`. I wonder if it's worth making this a union type (i.e. `number | string`). I've been using the type `SpaceProps["m"]` from styled-system in palette for these kinds of types but I don't know if you have access to that here.

File: src/lib/Scenes/City/CityBMWSponsorship.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+          tracking.trackEvent({
+            action_name: Schema.ActionNames.BMWLogo,
+            action_type: Schema.ActionTypes.Tap,
+          })
+        }}
+        ref={navRef}
+      >
+        <Flex flexDirection="row">
+          <Logo source={require("../../../../images/BMW_Grey-Colour_RGB.png")} />
```
Comment (xtina-starr) [inline src/lib/Scenes/City/CityBMWSponsorship.tsx:38]: Is relative path supported here like in reaction (i.e. images/BMW_Grey-Colour_RGB.png) or is that only when you are using the `import {...} from "<some file path>"` format??

**Pull request merged**